src/_pytest/config/__init__.py

```diff
@@ -651,6 +671,11 @@ def import_plugin(self, modname, consider_entry_points=False):
                 self.hook,
                 stacklevel=2,
             )
+        except Exception as e:
+            raise (PluginImportFailure(modname, sys.exc_info())).with_traceback(
```

Comment:
there's an extra set of parens here that aren't needed -- also black kinda mangled this didn't it :S

Reply:
Please see if you meant it (remove ``from e``)?

Reply:
I think @asottile meant to remove the parenthesis around `raise`: `raise (...)`